main.go

```diff
@@ -110,7 +112,17 @@ func registerServices(client clusterClient, grpcServer grpc.ServiceRegistrar) er
 		config.CurrentLokiGatewayConfig.Password,
 	)
 
-	pbLogging.RegisterDoguLogMessagesServer(grpcServer, logging.NewLoggingService(lokiLogProvider))
+	loggingService := logging.NewLoggingService(
+		lokiLogProvider,
+		cesReg,
+		doguinteraction.NewDefaultDoguInterActor(client, config.CurrentNamespace, cesReg),
+		local.NewCombinedLocalDoguRegistry(
+			client.CoreV1().ConfigMaps(config.CurrentNamespace),
```

Comment:
🚫 **[checkstyle]** <typecheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>client.CoreV1 undefined (type clusterClient has no field or method CoreV1)

Reply:
false positive as CoreV1 is exported in kubernertes.Interface which is used by the clientSet interface

Reply:
:eyes: :eyes: :eyes:

Reply:
False positive via golangci-lint. Was fixed by amending the respective version in the `Makefile`